app/posts/smoking-cessation/2025/03/2025-03-20-smoking-cessation-pilot.md

```diff
@@ -0,0 +1,143 @@
+
+Our planned research activity would allow us to:
+
+1. Test the efficacy of the incentive-based payment model for providers and evaluate the quality of their services
+2. Better understand the needs to user cohorts to ensure they receive a service that meets their needs and expectations
+3. Ensure NHSE capture the right data to determine outcomes are being met
+
+## What we learned
+
+Our research and design activities focused on leveraging existing data and conducting interviews with stakeholders to ensure we have a strong foundational knowledge of the existing service, before conducting primary research.
+
+Our understanding of the Digital Tobacco Treatment (DTT) service was informed by conversations with:
+- DTT team – collaborating with the tobacco team to understand the service proposition and map the end-to-end user journey
+- Service providers – to understand the service from the perspective of those providing it, visualising each step a user goes through when using the service
+- National Voices – drawing on insights from adjacent research, which focused on user experience when attempting to stop smoking with one of our service providers
+
+For both phases, we created user journeys and service maps which highlight the steps that will need to be taken by NHS Staff, NHSE colleagues and service providers for each of the phases to work seamlessly and help us ensure our research plan will validate or test key areas for discovery.
+
+![The user journey maps phases 1 and 2. There is a link provided to see the maps in a more navigable format.](user-journeys.png)
+
+[See the user journey map in Mural (needs an HSCIC account)](https://app.mural.co/t/nhsdigital8118/m/nhsdigital8118/1738339722878/729c1da461b58b6d192ffc44cb060c547424ed0f).
+
+After the first and second phase, there will be further iteration to the service to make sure we meet the desired outcomes for tobacco cessation.
+
+## Core elements of the current and future service vision
+
+We mapped the core elements of phase 1 and 2 against what a service for members of the public would need.
+
+The core elements we focused on were:
+- who this service will ultimately serve
+- how this would look and feel for users
+- who the service providers could be
+- the scope of the work (for example tobacco or tobacco and nicotine)
+- integration with wider NHSE services
+
+By having this articulated, we can let the research and insights guide us to how we should adapt the current service to meet this vision going forward.
+
+### Current vision:
+
+![This image shows six scales which relate to core elements of the current service vision. There are points to depict where along the scales they currently sit, along with a small descriptor. The aim is for this image to be compared with the future service vision to understand the differences between the two.](current-service-vision.png)
```

Comment:
Could these alts work a bit harder? Perhaps go with a figure and do more to write out what they're communicating?